src/common/l10n.c

```diff
@@ -34,20 +34,60 @@
 #include <windows.h>
 #endif
 
+static gchar* _dt_full_locale_name(const char *locale)
+{
+#ifdef __APPLE__
+  return dt_osx_full_locale_name(ui_lang);
+#endif
+#ifdef __linux__
+  gchar *output = NULL;
+  GError *error = NULL;
+  // This would likelly work on linux and macos
```

Comment:
likelly -> likely, but it does work on macOS, and I assume on most Linux installation too, so just remove this comment I think.

Reply:
OK, done :+1: